eng/Signing.props

```diff
@@ -121,4 +72,32 @@
     <FileSignInfo Include="Microsoft.Identity.Client.dll" CertificateName="None" />
   </ItemGroup>
 
+  <!-- $(InstallersOutputPath), $(SymbolsOutputPath), and $(ChecksumExtensions) are not defined. Root Directory.Build.props is not imported. -->
+  <ItemGroup>
+    <!-- Prepare for _PublishInstallersAndChecksums target. -->
+    <_InstallersToPublish Include="$(ArtifactsPackagesDir)**\*.jar" UploadPathSegment="jar/" Condition="'$(PublishAllBuildsAssetsInThisJob)' == 'true'" />
```

Comment:
just a nit: this item doesn't need to prefixed with a `_` here as there's not really a chance for a item conflict.

Reply:
It's a pattern we tend to follow in aspnetcore, the `_` prefix specifies a temporary/custom item